py/strato/common/log/configurelogging.py

```diff
@@ -114,16 +116,43 @@ def _configureOutputToFile(logger, logName):
     handler = logging.FileHandler(filename=logFilename(logName))
     atexit.register(handler.close)
     handler.setFormatter(machinereadableformatter.MachineReadableFormatter())
-    handler.set_name("file")
+    handlerName = "file_%s" % logName if logName != _name else "file"
+    handler.set_name(handlerName)
     handler.setLevel(logging.DEBUG)
     logger.addHandler(handler)
     logger.findCaller = _findCaller
 
-
-def _configureLogLevels():
+def _configureLogLevels(name):
     if config.LOG_CONFIGURATION is not None:
         with open(config.LOG_CONFIGURATION, 'rt') as f:
             dictConfig = json.load(f)
     else:
         dictConfig = config.DEFAULT_LOG_CONFIGURATION
+    if os.path.exists(config.LOGS_CONFIGURATION_OVERRIDE_FILE):
+        with open(config.LOGS_CONFIGURATION_OVERRIDE_FILE, 'rt') as f:
+            overrides = json.load(f)
+        default_overrides = overrides.get('default_log_overrides', {})
+        dictConfig.update(default_overrides)
+        dictConfig.update(overrides.get(name, {}))
     logging.config.dictConfig(dictConfig)
+
+def _reloadLoggingConfiguration(signal, stackFrame):
+    global _name
+    if _name is not None:
+        _configureLogLevels(_name)
+
+def _getMultipleFuncsHandler(funcs):
+    def _multipleFuncsHandler(signalNumber, stackFrame):
+        for func in funcs:
+            func(signalNumber, stackFrame)
+    return _multipleFuncsHandler
+
+def _configureLoggingSignalHandlers():
+    currentHandler = signal.getsignal(config.UPDATE_LOGGING_CONFIGURATION_SIGNAL)
```

Comment:
I think it can be poroblematic in case that we use oslo systemd services as an infrastructure (and we do use them in all datalayer projects). The problem is that logging must be imported and configred before any other imports, so this method that configures signals will be called before other signal handlers will register. 
Than when oslo.service signal handlers will register (we use systemd service wrapper)  it will erase the one that you register, since it does not support "nesting".

Good solution would be to expose both "register" and "reload" methods (without "_" :) ) so that every one will choose which one to use. maybe provide a "register" and a optional parameter to "configurelogging"?